CAIPs/caip-26.md

```diff
@@ -0,0 +1,69 @@
+---
+caip: 26
+title: Blockchain Reference for the Tezos Namespace
+author: Stanly Johnson (@stanly-johnson)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/36
+status: Draft
+type: Standard
+created: 2020-12-12
+updated: 2020-12-12
+requires: 2
+---
+
+## Simple Summary
+
+This document is about the details of the Tezos namespaces and references for CAIP-2.
+
+## Abstract
+
+In CAIP-2 a general blockchain identification scheme is defined. This is the
+implementation of CAIP-2 for Tezos.
+
+## Motivation
+
+See CAIP-2.
+
+## Specification
+
+### Tezos Namespace
+
+The namespace "tezos" refers to the Tezos open-source blockchain platform.
+
+#### Reference Definition
+
+The definition for this namespace will use the `genesis-block-hash` as an indentifier for different Tezos chains. The method for calculating the chain Id is as follows 
+```
+tezosB58CheckEncode('Net',
+  firstFourBytes(
+    blake2b(msg = tezosB58CheckDecode('B', genesisBlockHash),
+            size = 32)))
+```
```

Comment:
Hmm, four bytes is really small for avoiding collisions. Do you expect many tezos chains in the future?

I'm just curious. If the Tezos community is happy with this format, it's good for this CAIP as well.

Reply:
Tezos is not meant for a multi chains (like cosmos or polkadot), if thats what you mean. Beyond that I agree with your point. Maybe someone from the core team can comment.